contents/careers/senior-fs-software-engineer.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Senior Full Stack Software Engineer
+sidebar: Careers
+showTitle: true
+---
+**PostHog exists to increase the number of successful products in the world.** 
+
+We provide an open source platform for product analytics. The product is new - we launched in February 2020, and have had thousands of deployments organically since we started.
+
+This growth has meant we've been able to raise significant investment from the best in the world, such as YCombinator and GV. We are now looking to grow our engineering team and we are looking for **Senior Full Stack Engineers** to join us!
+
+**The release schedule is wild**
+
+We're up against billion dollar companies, and we're catching up. We will overtake them by being faster.
+
+[Comments like this](https://news.ycombinator.com/item?id=22378393) are what we live and die by.
```

Comment:
Is this a good comment? I just read it (want to send the offer to a friend I've been wooing since the beginning), but this feels just super meh. The "a feature request" given there is basically wrong and I wouldn't live and die by that.

Reply:
Said in a different way, up to this point the offer sounded very exciting, and while the first paragraph in the comment was awesome, I had a very low energy vibe after.

Reply:
Super happy to change it! Do you have one that works a bit better? @mariusandra

Reply:
Unfortunately not :(.